src/components/Cards/NotificationCard.tsx

```diff
@@ -28,41 +24,13 @@ function NotificationContent(props: NotificationContentDefinition) {
   return <span innerHTML={props.html} />;
 }
 
-const getNotificationContent = (type: string) => {
-  switch (type) {
-    case 'warning':
-      return warningContent;
-    case 'error':
-      return errorContent;
-    case 'info':
-    default:
-      return infoContent;
-  }
-};
-
-export const NotificationCard = () => {
-  const [showCard, setShowCard] = createSignal(true);
-  const showNotification = import.meta.env.VITE_SHOW_NOTIFICATION === 'true';
-  const [isInitialized, setIsInitialized] = createSignal(false); //
-  const isClient = typeof window !== 'undefined';
-
-  createEffect(() => {
-    if (isClient) {
-      const hasDismissedCard =
-        localStorage.getItem('notificationDismissed') === 'true';
-      if (hasDismissedCard) {
-        setShowCard(false);
-      }
-      setIsInitialized(true);
-    }
-  });
-
-  const handleDismiss = () => {
-    if (isClient) {
-      localStorage.setItem('notificationDismissed', 'true');
-    }
-    setShowCard(false);
-  };
+const NotificationCard = () => {
+  const [showCard, setShowCard] = createSignal(
+    localStorage.getItem('notificationDismissed') !== 'true'
+  );
+  const showNotification = JSON.parse(
+    import.meta.env.VITE_SHOW_NOTIFICATION || 'false'

```

Comment:
I'm guessing you're using JSON.parse to coerce the env to a boolean? if thats the case lets make 'false' just the boolean `false`